src/rpm/builder.rs

```diff
@@ -68,24 +65,21 @@ async fn async_file_mode(_file: &async_std::fs::File) -> Result<u32, RPMError> {
     Ok(0)
 }
 
-/// Returns a value that can be used for associating several package builds as being part of one operation
-///
-/// You can use any value, but the standard format is "${build_host} ${build_time}"
-///
-/// See `RPMBuilder::cookie()`
```

Comment:
Move this description to `RPMBuilder::cookie()` please.

https://github.com/rpm-rs/rpm/blob/815ddb38afef051c2e3dfb0cf8eae99f5c96aa32/src/rpm/builder.rs#L205-L212